src/auth/base-auth.ts

```diff
@@ -827,6 +827,35 @@ export abstract class BaseAuth {
     return this.authRequestHandler.getEmailActionLink('VERIFY_EMAIL', email, actionCodeSettings);
   }
 
+  /**
+   * Generates the out of band email action link to verify the user's ownership
+   * of the specified email. The {@link ActionCodeSettings} object provided
+   * as an argument to this method defines whether the link is to be handled by a
+   * mobile app or browser along with additional state information to be passed in
+   * the deep link, etc.
+   *
+   * @param email - The current email account.
```

Comment:
address

Reply:
Kept it as email account as the previous public method (verify_email, reset_password etc) uses the same documentation on email paramater.  Let me know if you feel strong about `email address`.